cli/commands/test.py

```diff
@@ -35,59 +38,72 @@ def main(
         case_sensitive=False,
         default=None,
     ),
-    content_type: Optional[ContentType] = typer.Option(
+    content_type: ContentType = typer.Option(
         help="The content type to run tests for",
-        case_sensitive=False,
         show_choices=True,
+        case_sensitive=False,
+        prompt=True,
         default=None,
     ),
-    target: Optional[Target] = typer.Option(
-        help="The target to run tests against",
-        case_sensitive=False,
-        show_choices=True,
-        default="development",
+    query: Optional[str] = typer.Option(
+        help="The query to test: a string of JSON, a local file path, or a URL of catalogue API search templates",
```

Comment:
can a user actually supply a json string query with the implementation below? Admittedly I haven't tried running it, but It looks like we have one block for loading a query from an api search template endpoint, and two for loading from a local file

Reply:
Ah yes, this was an initial idea but it made the code a bit mad and is probably useless - will remove it